components/_modals.scss

```diff
@@ -0,0 +1,118 @@
+/*
+
+Stylesheet for all things related to presentation of 
+modals across the website.
+
+*/
+
+// Overlay behind the modal
+
+.ui-widget-overlay {
+	background: $tws-yellow;
+}
+
+// General modal box positioning
+
+.ui-dialog {
+
+	position: fixed !important;
+  top: 50% !important;
+  left: 50% !important;
+  -webkit-transform: translate(-50%, -50%) !important;
+  -moz-transform:    translate(-50%, -50%) !important;
+  -ms-transform:     translate(-50%, -50%) !important;
+  -o-transform:      translate(-50%, -50%) !important;
+  transform:         translate(-50%, -50%) !important;
+  padding: 0;
+  overflow: visible;
+
+  // Eliminating borders and placing cancelation X properly
+
+  .ui-dialog-titlebar {
+
+  	padding: 0;
+  	top: 0;
+  	right: 0;
+  	position: absolute;
+
+  	// Cancel button
+
+  	.ui-dialog-titlebar-close {
```

Comment:
Line should be indented with spaces, not tabs<br>Selector should have depth of applicability no greater than 2, but was 3